Servers/UI/OJS.Servers.Ui.Models/Submissions/Details/SubmissionDetailsResponseModel.cs

```diff
@@ -36,5 +37,7 @@ public class SubmissionDetailsResponseModel : IMapFrom<SubmissionDetailsServiceM
         public DateTime CreatedOn { get; set; }
 
         public DateTime? ModifiedOn { get; set; }
+
+        public ValidationResult ValidationResult { get; set; } = null!;
```

Comment:
Why not make ValidationResult nullable instead of doing "null!"?

"Basically, null! applies the ! operator to the value null. This overrides the nullability of the value null to non-nullable, telling the compiler that null is a "non null" type."

Reply:
I have never thought of it this way. ValidationResult is now nullable.